app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] [REQUIRED] Good job so far! The `associations` between `models` should be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Please implement my suggestions below 🙏 

```ruby
# app/models/comment.rb
belongs_to :author, class_name: 'User'
belongs_to :post
```